src/libraries/System.Text.RegularExpressions/README.md

```diff
@@ -0,0 +1,15 @@
+# System.Text.RegularExpressions
+This assembly provides regular expression functionality that may be used from any platform or language that runs within .NET. The main type contained in the assembly is [Regex](https://learn.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex) class, which represents an immutable regular expression object which can be used to test for matches on given input strings or spans.
```

Comment:
@IEvangelist, is this common wording in our docs?  What does it mean for a platform to run within .NET?

Reply:
@stephentoub, no. That isn't common wording in our docs. Perhaps the language part, since there are multiple languages within .NET, but not platforms. Perhaps workloads would be more appropriate here, or just remove the platform part of it?

> This assembly provides regular expression functionality that may be used from any language that runs within .NET. The main type contained in the assembly is the [Regex](https://learn.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex) class, which represents an immutable regular expression object is used to test for matches on given input strings or spans.

> **Note**
> In my rewording above, I also updated the text to be active rather than passive. This is preferred in our docs as well.

Reply:
Thanks. I'm also not sure what it means for a language to run within .NET. :)

For both this readme and the docs, I'd suggest simplifying it to just "regular expression functionality for .NET".  Or just "regular expression functionality", since everything being discussed is in the context of .NET.